calculator/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <div class="calculator">
+            <div class="calculator-display">
+                <input class="display" type="text" disabled value="0">
+            </div>
+            <div class="calculator-buttons">
+               <button id="ce" class="button clear-btn">CE</button>
+               <button id="c" class="button clear-btn">C</button>
```

Comment:
Tip: Always specify the type attribute for a <button> element, to tell browsers what type of button it is.
https://www.w3schools.com/tags/tag_button.asp

Reply:
Thanks for the advice, I'll fix it